src/main/java/com/spikes2212/command/drivetrains/commands/FollowPath.java

```diff
@@ -23,24 +23,26 @@
     private PIDController leftController;
     private PIDController rightController;
 
-    public FollowPath(OdometryDrivetrain drivetrain, Path path, double lookaheadDistance,
-                      PIDSettings pidSettings, FeedForwardSettings feedForwardSettings, double maxAcceleration) {
+    public FollowPath(OdometryDrivetrain drivetrain, List<Waypoint> path, double lookaheadDistance,
+                      PIDSettings pidSettings, FeedForwardSettings feedForwardSettings, double maxAcceleration,
+                      boolean inverted) {
         addRequirements(drivetrain);
         this.drivetrain = drivetrain;
         this.path = path;
         this.lookaheadDistance = lookaheadDistance;
         this.pidSettings = pidSettings;
         this.FeedForwardSettings = feedForwardSettings;
         this.maxAcceleration = maxAcceleration;
+        drivetrain.setInverted(inverted);
     }
 
     @Override
     public void initialize() {
         drivetrain.zeroSensors();
         purePursuitController = new PurePursuitController(drivetrain.getHandler(), path,
-                lookaheadDistance,  maxAcceleration, drivetrain.getWidth());
-        purePursuitController.getOdometryHandler().set(purePursuitController.getPath().getPoints().get(0).getX(),
-                purePursuitController.getPath().getPoints().get(0).getY());
+                lookaheadDistance, maxAcceleration, drivetrain.getWidth());
+        purePursuitController.getOdometryHandler().set(purePursuitController.getPath().get(0).getX(),
```

Comment:
you may add a utility method to `PurePursuitController` that returns the initial location of the path it is suppose to follow. thus, you'll save youself this large `OdometryHandler.set` call.